test/kind/init/tests/00-install.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+  - command: docker cp kind-control-plane:/etc/kubernetes/pki/ca.crt /tmp/ca.crt
+  - command: docker cp kind-control-plane:/etc/kubernetes/pki/ca.key /tmp/ca.key
+  - command: kubectl create namespace cert-manager
+  - command: kubectl create secret tls kubernetes-root-ca --namespace cert-manager --cert=/tmp/ca.crt --key=/tmp/ca.key
+  - command: kubectl apply -f ../../../../addons/konvoyconfig/0.0.x/konvoyconfig-1.yaml
+  - command: kubectl apply -f ../../../../addons/cert-manager/0.10.x/cert-manager-4.yaml
+  - command: ./metallb-config.sh
+  - command: kubectl apply -f ../../../../addons/metallb/0.8.x/metallb-1.yaml
```

Comment:
you may need order (meaning `metallb-config.sh` may need cert-manager-4.yaml prior to it being called or metallb-1.yaml needs to follow apply)... and if order control is needed this is a good way.

for "kubectl apply -f" that can come after commands you can now with `v0.3.0` do the following:

```
apply:
  - ../../../../addons/cert-manager/0.10.x/cert-manager-4.yaml
  - ../../../../addons/metallb/0.8.x/metallb-1.yaml

```